.github/workflows/ensure-cms-branch.yml

```diff
@@ -0,0 +1,30 @@
+name: Ensure correct CMS branch
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    paths:
+      - src/admin/config.yml # Skip this action unless the Netlify CMS config file has changed.
+
+jobs:
+  ensure-cms-branch:
+    runs-on: ubuntu-latest
+    env:
+      target_branch: "dev" # Default to dev branch
+    steps:
+    - uses: actions/checkout@v2
+    - name: Adjust target branch for environment
+      if: endsWith(github.ref, "/main") # If the commit is in
```

Comment:
Sorry about my ignorance about the syntax of Github Actions. It seems [strings have to use single quotes](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals). Using double quotes [breaks the job](https://github.com/cindyli/wecount.inclusivedesign.ca/actions/runs/741876536).

Reply:
Oh, interesting—I wasn't aware of that!